cfme/control/explorer/alerts.py

```diff
@@ -256,7 +257,14 @@ def _fill(self, view):
             else:
                 fill_details["snmp_trap_send"] = True
                 fill_details["snmp_trap"] = self.snmp_trap
-        view.fill(fill_details)
+        if self.emails is not None:
+            if not self.emails:
```

Comment:
actually, you check self.emails 2 times for the same. I guess one of if statements can be removed